db/clean_planting_csv/clean_planting_API_call.py

```diff
@@ -0,0 +1,97 @@
+import requests, zipfile, io
+import json
+import pandas as pd
+import uuid
+
+central_url = "https://senecaparkzoo.getodk.cloud"
```

Comment:
So there's this python convention that if a file is meant to be run form the command line like a script, all code should be wrapped in a function. The only code that isn't in a function is the following

```python
if __name__ == "__main__":
    run_script() # function that runs your script
```

Reply:
Ok, will make that change.